console/src/user/RegisterModal.tsx

```diff
@@ -0,0 +1,170 @@
+      if (!methods.validate()) return;
+      const values = methods.value();
+      if (client == null) throw new Error("No Cluster Connected");
+      await client.user.create({ ...values });
+      onClose();
+    },
+    onError: (e) =>
+      addStatus({
+        message: "Failed to register user",
+        description: e.message,
+        variant: "error",
+      }),
+  });
+
+  return (
+    <Align.Space style={{ paddingTop: "2rem", height: "100%" }} grow empty>
+      <Align.Space
+        className="console-form"
+        justify="center"
+        style={{ padding: "1rem 3rem" }}
+        grow
+      >
+        <Form.Form {...methods}>
+          <Align.Space direction="y">
+            <Form.Field<string> path="username">
+              {(p) => (
+                <Input.Text
+                  autoFocus
+                  level="h2"
+                  variant="natural"
+                  placeholder="username"
+                  {...p}
+                />
+              )}
+            </Form.Field>
+            <Form.Field<string> path="password">
+              {(p) => (
+                <Input.Text
+                  level="h2"
+                  variant="natural"
+                  placeholder="password"
+                  type="password"
+                  {...p}
+                />
+              )}
+            </Form.Field>
+            <Align.Space direction="x">
+              <Form.Field<string> path="firstName" label="First Name">
+                {(p) => (
+                  <Input.Text
+                    level="h3"
+                    variant="natural"
+                    placeholder="Richard"
+                    {...p}
+                  />
+                )}
+              </Form.Field>
+              <Form.Field<string> path="lastName" label="Last Name">
+                {(p) => (
+                  <Input.Text
+                    level="h3"
+                    variant="natural"
+                    placeholder="Feynman"
+                    {...p}
+                  />
+                )}
+              </Form.Field>
+            </Align.Space>
+          </Align.Space>
+        </Form.Form>
+      </Align.Space>
+      <Nav.Bar location="bottom" size={48}>
+        <Nav.Bar.Start style={{ paddingLeft: "2rem" }} size="small">
+          <Triggers.Text shade={7} level="small" trigger={SAVE_TRIGGER} />
+          <Text.Text shade={7} level="small">
+            To Register
+          </Text.Text>
+        </Nav.Bar.Start>
+        <Nav.Bar.End style={{ paddingRight: "2rem" }}>
+          <Button.Button
+            onClick={() => mutate()}
+            disabled={client == null || isPending}
+            tooltip={
+              client == null
+                ? "No Cluster Connected"
+                : `Save to ${client.props.name ?? "Synnax"}`
+            }
+            tooltipLocation="bottom"
+            loading={isPending}
+            triggers={[SAVE_TRIGGER]}
+          >
+            Register in Synnax
```

Comment:
Why is this called "Register in Synnax", where else would you be registering a user to? Just call it `Register`